app/Services/AgendaApplicationFormService.php

```diff
@@ -142,6 +144,10 @@ public function getExportData(AgendaItem $agendaItem): Collection
             foreach ($selectedElements as $element) {
                 $userline[$element] = $user[$element];
             }
+            
+            //format birthday to not include the time of day
+            $userline["birthDay"] = substr($userline["birthDay"], 0, 10);
```

Comment:
I'd suggest performing this attribute formatting in the `User` model instead. There, we can firsts add
```php
protected $casts = [
    'birthDay' => 'date',
];
```
This will ensure the `birthDay` attribute is always type-casted to a [Carbon](https://carbon.nesbot.com/docs/) instance.

Then, I'd suggest defining an [accessor](https://laravel.com/docs/10.x/eloquent-mutators#defining-an-accessor) to retrieve the "formatted birthday", as follows.
```php
public function birthdayFormatted(): Attribute
{
    return Attribute::make(
        get: fn () => $this->birthDay->format('Y-m-d'),
    );
}
``` 

Note that the formatted birthday value can now simply be retrieved using `$user->birthdayFormatted` (not calling the function, but relying on the new property that will be magically added by the framework) or `$user['birthdayFormatted']` whenever we have an instance of the user model.